tests/bdp_service_manager.erl

```diff
@@ -0,0 +1,139 @@
+%% -------------------------------------------------------------------
+%%
+%% Copyright (c) 2015 Basho Technologies, Inc.
+%%
+%% This file is provided to you under the Apache License,
+%% Version 2.0 (the "License"); you may not use this file
+%% except in compliance with the License.  You may obtain
+%% a copy of the License at
+%%
+%%   http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing,
+%% software distributed under the License is distributed on an
+%% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+%% KIND, either express or implied.  See the License for the
+%% specific language governing permissions and limitations
+%% under the License.
+%%
+%% -------------------------------------------------------------------
+%% @doc A module to test BDP service manager operations: add a
+%%      service, then perform a start/stop cycle on various
+%%      combinations of starting, stopping and execution nodes
+
+-module(bdp_service_manager).
+-behavior(riak_test).
+-export([confirm/0]).
+-include_lib("eunit/include/eunit.hrl").
+
+-define(S1_NAME, "service-one").
+-define(S1_TYPE, "cache-proxy").
+%% (this is parameter 'Group' in various calls to data_platform_global_state functions)
```

Comment:
Comment seems a bit misleading. It's referring tot he line above, rather than the line below. It also seems to indicate the macro is used as the group name _instead of_ rather than _in addition to_ the service type.